substratevm/DEBUGINFO_WINDOWS.md

```diff
@@ -0,0 +1,35 @@
+Using the prototype debug info feature on windows.
+--------------------------------------
+
+To add debug info to a generated native image add flag
+-H:GenerateDebugInfo=<N> to the native image command line (where N is
+a positive integer value -- the default value 0 means generate no
+debug info). For example,
+
+    $ javac Hello.java
+    $ mx native-image -H:GenerateDebugInfo=1 Hello
+
+The resulting image should contain CodeView4 debug records in a
+format Visual Studio understands. 
+
+Please read the standard DEBUGINFO file; this file only documents differences.
+
+Because of limitations in the linker, function names are mangled into the equivalent of
+ _package.class.Function_999_, where '999' is a hash of the function arguments.  The exception is the first main() function encountered,
+ which is mangled to _package.class.main_, with no argument hash.
+
+As an experimental feature, the Windows debug info ignores inlined Graal code.
```

Comment:
Ok, I misread this at first. I thought you were configuring this option on the native image command line. I see now this mode can only be enabled by editing the source for CVConstants and rebuilding a GraalVM distro.
I guess it's ok to retain this mode for experimentation but I don't think it is useful to document it in the .md file for users.

Reply:
I need to find out a good method of naming Java functions so they look obvious and (ideally) are not too hard to type.  Perhaps the mangled names similar to (bar underscores) how javah would output them?  Unfortunately, to do this I'd need access to the type information, which wasn't available at the time of coding, but I think DebugInfo has it now, or will soon.

But, your comment is about the inlined Graal code.  I'll remove that from the docs, and address the code in the other review items.

Reply:
What type info do you need? Can you not process the method signature into a mangled name?

Reply:
Do argument type names contain package names?  I am currently thinking I'll just escape characters that the Microsoft linker doesn't understand, but long term I'm hoping for something more precise.  I'll leave it as a hash of the signature for now, and either do a separate PR or a separate commit on this PR to make it more readable.

Reply:
Yes arg type names are package qualified. A follow up patch should be ok.

Reply:
Agreed.  I want to be slightly more clever here later on.